packages/kununu-utils/kununu-logger/index.js

```diff
@@ -24,22 +30,22 @@ export const logLevelNum = {
  * @returns {string} Stringified object
  */
 export const formatNodeRequest = ({
```

Comment:
Should this be in another file? This file is importing `fingers-crossed` and `fingers-crossed` is importing this file...